### PR TITLE
doc: I had to read this twice before understanding it

### PR DESCRIPTION
**Pull request by tshepang**

### CODE REVIEW

File: src/libcore/default.rs [hunk 1]
```diff
@@ -78,7 +78,7 @@
 
 #![stable(feature = "rust1", since = "1.0.0")]
 
-/// A trait that types which have a useful default value should implement.
+/// A trait that must be implememnted by types that have a useful default value.
```
Comment (nagisa) [inline src/libcore/default.rs]: Why “must”? We do not enforce this invariant, nor will anything break due to breaking this now-a-requirement.

Comment (steveklabnik) [inline src/libcore/default.rs]: Yes, this is wrong. What about

> A trait for giving a type a useful default value.

Comment (tshepang) [inline src/libcore/default.rs]: done

**Pull request merged**